ui/application-window.ui

```diff
@@ -49,6 +49,7 @@ Author: Philipp Wolfer <ph.wolfer@gmail.com>
     <property name="icon_name">com.uploadedlobster.peek</property>
     <property name="show_menubar">False</property>
     <signal name="screen-changed" handler="on_window_screen_changed" swapped="no"/>
+    <signal name="draw" handler="on_window_draw" swapped="no"/>
```

Comment:
Ahhh... I was looking for such a file (I failed to find the definitions of the various widget), but I was looking only inside of `src/ui/`. This explains why I did not see it. :-)